lib/api/apiUtils/object/corsResponse.js

```diff
@@ -0,0 +1,130 @@
+/** _matchesValue - compare two values to determine if they match
+* @param {string} allowedValue - an allowed value in a CORS rule;
+    may contain wildcards
+* @param {string} value - value from CORS request
+* @return {boolean} - true/false
+*/
+function _matchesValue(allowedValue, value) {
+    const wildcardIndex = allowedValue.indexOf('*');
```

Comment:
No, there is only one asterisk permitted per value and that should have been validated when the cors rules were processed in putBucketCors. :)